Cargo.toml

```diff
@@ -1,42 +1,8 @@
-[package]
-name = "dropshot"
-description = "expose REST APIs from a Rust program"
-version = "0.1.0"
-authors = ["David Pacheco <dap@oxidecomputer.com>"]
-edition = "2018"
-license = "Apache-2.0"
-repository = "https://github.com/oxidecomputer/dropshot/"
-
-[dependencies]
-async-trait = "0.1.24"
-bytes = "0.5.4"
-chrono = { version = "0.4", features = [ "serde" ] }
-dropshot_endpoint = { path = "dropshot_endpoint" }
-futures = "0.3.1"
-http = "0.2.0"
-hostname = "0.3.0"
-hyper = "0.13"
-indexmap = "1"
-openapiv3 = "0.3"
-schemars = "0.7"
-serde = { version = "1.0", features = [ "derive" ] }
-serde_json = "1.0"
-serde_urlencoded = "0.6"
-slog-async = "2.4"
-slog-bunyan = "2.2"
-slog-json = "2.3"
-slog-term = "2.5"
-tokio = { version = "0.2", features = [ "full" ] }
-toml = "0.5.6"
-uuid = { version = "0.8", features = [ "serde", "v4" ] }
-
-[dev-dependencies]
-libc = "0.2.71"
-
-[dependencies.slog]
-version = "2.5"
-features = [ "max_level_trace", "release_max_level_debug" ]
-
+#
+# Dropshot is comprised of two crates.  Consumers only need to know about
```

Comment:
```suggestion
# Dropshot is comprised of two packages.  Consumers only need to know about
```

This is not the most important difference, but while I'm here...

Reply:
Interesting!  Thanks for pointing it out.  I just looked up the definitions in the [Cargo book glossary](https://doc.rust-lang.org/cargo/appendix/glossary.html), and now it feels awfully confusing that it's called `crates.io`!

Reply:
Fixed in 584593e8a05e7a80e9b1de1d6dd7022883156dab.